Sources/NostrSDK/Events/Calendars/TimeOmittedDate.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  TimeOmittedDate.swift
+//
+//
+//  Created by Terry Yiu on 11/19/23.
+//
+
+import Foundation
+
+/// A representation of date as it would appear in ISO 8601 format, which is in the Gregorian calendar. Time and time zone are omitted.
+public struct TimeOmittedDate: Comparable {
+    public static func < (lhs: TimeOmittedDate, rhs: TimeOmittedDate) -> Bool {
+        if lhs.year < rhs.year {
+            return true
+        }
+
+        if lhs.year > rhs.year {
+            return false
+        }
+
+        if lhs.month < rhs.month {
+            return true
+        }
+
+        if lhs.month > rhs.month {
+            return false
+        }
+
+        return lhs.day < rhs.day
```

Comment:
ChatGPT suggests this simpler alternative:
```swift
if lhs.year != rhs.year {
        return lhs.year < rhs.year
    }

    if lhs.month != rhs.month {
        return lhs.month < rhs.month
    }

    return lhs.day < rhs.day
```

Reply:
Much nicer. Fixed!